scaffold/github/workflows/ComposerLockDiff.yml

```diff
@@ -3,6 +3,8 @@ name: "Composer Lock Diff"
 on:
   pull_request:
     types: [opened, synchronize, reopened]
+    paths:
+      - 'composer.lock'
```

Comment:
I had to look this up. This is a really smart simple solution to only execute this workflow if the file had changed. Nice!

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore